packages/calcite-components/package.json

```diff
@@ -26,21 +26,21 @@
     "build-storybook": "npm run util:build-docs && NODE_OPTIONS=--openssl-legacy-provider build-storybook --output-dir ./docs --quiet",
     "clean": "npm run util:clean-js-files && npm run util:clean-readmes && rimraf node_modules dist www hydrate docs .turbo",
     "deps:update": "updtr --exclude chalk cheerio typescript @types/jest jest jest-cli ts-jest @whitespace/storybook-addon-html && npm audit fix",
-    "docs": "build-storybook",
-    "docs:preview": "npm run util:build-docs && NODE_OPTIONS=--openssl-legacy-provider STORYBOOK_SCREENSHOT_LOCAL_BUILD=true start-storybook",
+    "screenshot-test": "build-storybook",
```

Comment:
This name reads as if it will run the screenshot tests. How about `screenshot-tests` and `screenshot-tests:preview`?